frontend/packages/console-shared/src/components/dashboard/status-card/states.tsx

```diff
@@ -29,7 +30,7 @@ export const healthStateMapping: { [key in HealthState]: HealthStateMappingValue
     priority: 1,
     health: HealthState.UNKNOWN,
     icon: <GrayUnknownIcon title="Unknown" />,
-    message: 'Unknown',
+    message: i18next.t('dashboard~Unknown'),
```

Comment:
can you add i18n for all states ?
I think we should convert healthStateMapping to function to make sure we get the correct translation when user changes the language

Reply:
Sure, sounds fair! We can change this map -> function
This will need some adjustments at other places therefore I would raise another PR for ease in handling.

Reply:
Removed this chnage from this PR

Reply:
And Created another PR https://github.com/openshift/console/pull/7372